.github/workflows/build-and-publish.yml

```diff
@@ -0,0 +1,31 @@
+name: Build and publish
+
+on: push
```

Comment:
This would publish `vosk-cli` on pypi every time someone pushes to this repository, potentially changing existing versions. Wouldn't it make more sense to only run this on tags?

Reply:
The build will be triggered on any push, but the actual publishing will only happen when the pushed commit is tagged (see line 27). Even then it will not upload to PyPI as long as the name of the built package did not change (for it to change you would need to declare a new version or name in the `setup.py` file - I believe existing PyPI versions cannot be changed).
But I can and will change this so the whole workflow will only run on tagged commits.